sdl-core/src/main/scala/io/smartdatalake/app/GlueSmartDataLakeBuilder.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2023 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.smartdatalake.app
+
+import io.smartdatalake.app.DatabricksSmartDataLakeBuilder.parser
+import io.smartdatalake.config.ConfigurationException
+import scopt.OParser
+
+/*
+ * AWS Glue Smart Data Lake Command Line Application.
+ *
+ * Glue passes all job parameters (command line arguments) down to the application. This application filters out the
+ *   Glue specific ones.
+ * SCOPT issue:
+ * This was tested with Glue 4.0. Glue 4.0 uses an older version of scopt, which is specified in the XBOOTPATH.
+ * To fix the issue:
+ *  - place the newer scopt jar an s3 bucket,
+ *  - specify the path to it in the job property `Dependent JARs path` and
+ *  - Job parameter: `--user-jars-first` with value `true`
+ */
+class GlueSmartDataLakeBuilder extends SmartDataLakeBuilder {
```

Comment:
Do you know if these arguments are specific to a certain version of glue or if they are expeted to remain stable?

Reply:
I'm talking about the Glue arguments.
The ones with --user-jars-first you can always change however you want.
Let's see how stable it is.
If glue keeps creating new arguments that we have to filter you then it may be worth extracting the arguments to ignore into some conf file. But for now it's not worth it, let's keep il like this

Reply:
the setting actually only causes scopt to issue warnings instead of error and abort as soon as something unexpected is passed, so the user still sees if we mistyped something